### ISSUE TITLE
Does not handle OperationTypes

### ISSUE THREAD

**Issue by melinath** [ivelum/graphql-py#6]
It looks like this library doesn't support having operation names as part of the query. For example, this works:

```
{
    hero {
        name
    }
}
```

while this does not:

```
query {
    hero {
        name
    }
}
```

Both should be valid according to the spec.

**Comment by melinath** [comment]
I looked into this a little more and it looks like the lexer works fine; the problem is in the parser.

**Comment by stebunovd** [comment]
fixed in version 0.7.1, already on PyPI. The problem was caused by the lack of support for anonymous queries. Thank you for flagging this!
